ngx-fudis/projects/ngx-fudis/src/lib/components/form/guidance/guidance.component.html

```diff
@@ -85,13 +85,15 @@
       <ng-container *ngIf="helpText">{{ helpText }}</ng-container>
       <ng-container *ngIf="control && maxLength">
         <span class="fudis-visually-hidden"
-          >{{ control.value?.length || 0 }}/{{ maxLength }} {{ _maxLengthText | async }}</span
+          >&nbsp; {{ control.value?.length || 0 }}/{{ maxLength }}
```

Comment:
It seems that we have two different implementations for reading the maxLength (`LocalizedTextGroup` vs. basic `TextArea`).

In TextArea it is read **within** the input helpText where as in LocalizedTexGroup it is read **after** helpText. Since it probably would be best to keep the flow same, the easiest way would be deleting this whole `ng-container`... (continued in next comment)